modules/javafx.graphics/src/main/native-prism-d3d/D3DLight.h

```diff
@@ -26,28 +26,26 @@
 #ifndef D3DLIGHT_H
 #define D3DLIGHT_H
 
-#include "D3DContext.h"
-
 // See     Light.h
 
 class D3DLight {
 public:
-    D3DLight();
-    virtual ~D3DLight();
+    D3DLight() = default;
+    virtual ~D3DLight() = default;
```

Comment:
It doesn't seem like this class is supposed to be subclassable. I would suggest removing the constructor and destructor declarations and marking the class `final`.

Reply:
It might be subclassed in the future. One of the next changes will be a performance upgrade attempt, and we might need to separate the light types instead of bundling them into one that simulates the others.

In theory, this class can be removed even and instead the Java code could call the rendering pipeline directly with all the needed parameters. It's a more intrusive change though, and might as well wait for Panama with this one.